src/configurations/adknowledgeportal/synapseConfigs/studies.ts

```diff
@@ -101,8 +102,19 @@ export const studiesDetailsPageProps: DetailsPageProps = {
         {
           name: 'Markdown',
           title: 'Acknowledgement',
-          columnName: 'Acknowledgement',
-          props: {},
+          standalone: true,
+          props: {
+            ownerId:'syn12666371',
+            wikiId:'617506'
+          },
+        },
+        {
+          name: 'MarkdownCollapse',
+          columnName: 'ackContext',
```

Comment:
And show the wiki referenced in ackContext in a MarkdownCollapse section.  Include the Copy link, and change the default text description